src/renderer/components/+cluster/cluster-overview.scss

```diff
@@ -7,7 +7,9 @@
   display: grid;
   grid-gap: $gridGap;
   grid-template-columns: repeat(auto-fit, minmax(500px, 1fr));
-  grid-template-rows: 1fr 1fr;
+  grid-template-rows:
```

Comment:
@Nokel81 Please let me know what do you think about this. I tried to name grid row areas